backend/workflow_manager/file_execution/file_execution_helper.py

```diff
@@ -0,0 +1,89 @@
+from typing import Optional
+
+from workflow_manager.file_execution.models import WorkflowFileExecution
+from workflow_manager.workflow_v2.enums import ExecutionStatus
+from workflow_manager.workflow_v2.models.execution import WorkflowExecution
+
+
+class FileExecutionHelper:
```

Comment:
NIT: @muhammad-ali-e since all these methods are closely tied to the `WorkflowFileExecution` model - do you think they should be moved to `.models.py` as instance methods of the model or the model manager?